extensions/azdata/src/api.ts

```diff
@@ -45,47 +45,47 @@ export function getAzdataApi(localAzdataDiscovered: Promise<IAzdataTool | undefi
 	return {
 		arc: {
 			dc: {
-				create: async (namespace: string, name: string, connectivityMode: string, resourceGroup: string, location: string, subscription: string, profileName?: string, storageClass?: string) => {
+				create: async (namespace: string, name: string, connectivityMode: string, resourceGroup: string, location: string, subscription: string, profileName?: string, storageClass?: string, additionalEnvVars?: { [key: string]: string }) => {
```

Comment:
Let's make a type for the env vars : 

`export type AdditionalEnvVars = { [key: string]: string};`

and put that in azdata-ext.d.ts

Reply:
done